### ISSUE TITLE
tools.selTournament number of cores

### ISSUE THREAD

**Issue by sh0gg0th** [DEAP/deap#573]
Does it use one core only? I am able to successfully parallelize  my `evaluate` using multiprocessing and see that normally all cores are utilized, but there are periods proportional to `tournsize` of my `tools.selTournament` where only one core is working.

Thanks!

**Comment by fmder** [comment]
Yes it uses only one core. We decided not to parallelize tournament selection because it should not take very long to complete anyway. How long is it in your case?

**Comment by sh0gg0th** [comment]
@fmder With my current setup, it takes much longer than the fitness function, I have a population of 100,000 and a tournament size of 5,000.
